### PR TITLE
Update snapshots

### PR DESCRIPTION
**Pull request by fregante**
Following https://github.com/sindresorhus/eslint-plugin-unicorn/commit/fe4e7bcfa6d427f4abc4e94446a553f0a68b1fa1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
 		"fix": "run-p --continue-on-error fix:*",
 		"fix:eslint-docs": "eslint-doc-generator",
 		"fix:js": "npm run lint:js -- --fix",
+		"fix:snapshots": "ava --update-snapshots",
```
Comment (fregante) [inline package.json:25]: I'm also including the snapshots in the `fix` command, which I thought was enough initially but it did not update the snapshots

**Comment by fregante** [comment]
@fisker vscode's integration test has been failing for a few days, but the affected haven't changed in years.

https://github.com/microsoft/vscode/blob/main/src/vs/workbench/services/search/test/node/fixtures/examples/NullPoinderException.js#L10

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
> but the affected haven't changed in years.

I'm not sure how the parser changed, but the typescript parser is super forgiving, it allow many kind of invalid syntax, it's been problem for years in Prettier to handle all  unexpected AST, until  last year they starts to add extra checks for invalid syntaxes. [My  PRs](https://github.com/typescript-eslint/typescript-eslint/commits?author=fisker)

**Comment by fisker** [comment]
Ah, I believe the parser is correctly treat is as module now, it enbles strict mode, which also request by me.. https://github.com/typescript-eslint/typescript-eslint/issues/9101

**Pull request merged**